src/main/java/core/basesyntax/ColorCreator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface ColorCreator {
+    Color getColor();
```

Comment:
make it as a simple getter inside of abstract class

Reply:
Changed